electrum/gui/kivy/uix/dialogs/tx_dialog.py

```diff
@@ -181,6 +182,31 @@ def update(self):
             self.fee_str = _('unknown')
             self.feerate_str = _('unknown')
         self.ids.output_list.update(self.tx.outputs())
+
+        def text_format(addr):
+            """
+            Chooses the appropriate text color and background color to 
+            mark receiving, change and billing addresses.
+
+            Returns: color, background_color
+            """
+            
+            # modified colors (textcolor, background_color) from electrum/gui/qt/util.py
+            GREEN = ("#000000", "#8af296")
+            YELLOW = ("#000000", "#ffff00")
+            BLUE = ("#000000", "#8cb3f2")
+            DEFAULT = ('#ffffff', '#4c4c4c')
+
+            colors = DEFAULT
+            if self.wallet.is_mine(addr):
+                colors = YELLOW if self.wallet.is_change(addr) else GREEN
+            elif self.wallet.is_billing_address(addr):
+                colors = BLUE
+            return (get_color_from_hex(color) for color in colors)
+
+        for data in self.ids.output_list.data:
+            data['color'], data['background_color'] = text_format(data['address'])
```

Comment:
It would probably be better to choose a different variable name.

Reply:
Fixed with commit https://github.com/roth-a/electrum/commit/67bab545635798f3f413d1725bcb1d912eb81b61